src/yargs-options.js

```diff
@@ -37,6 +37,12 @@ const options = {
     type: 'boolean',
     default: false,
   },
+  'privileged-addon': {
```

Comment:
I'd name this flag `privileged` only, it is shorter and the linter CLI is already called "addons-linter".

Reply:
I'm going to rename the option in the next round of changes as we agreed.

Reply:
Option renamed by rebasing this PR on top of #4326.